x/incentive/keeper/keeper_test.go

```diff
@@ -156,20 +156,21 @@ func (suite *KeeperTestSuite) TestGetSetSwapRewardIndexes() {
 					RewardFactor:   d("0.04"),
 				},
 			},
+			panics: true,
 		},
 		{
-			// this test is to detect any changes in behavior, it would be nice if Set didn't panic
-			name:     "setting empty indexes panics",
+			// this test is to detect any changes in behavior
```

Comment:
Yeah, nothing depended on `Set` panicking. I think I added these tests just to remember the slightly unexpected behaviour.

There's a check in `AccumulateSwapRewards` to avoid setting empty reward indexes that doesn't strictly need to be there now. But probably easier to leave in place.